2-mandatory/1-create-functions.js

```diff
@@ -44,7 +48,8 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(arr) {
+  return arr.map((n) => n > 100 ? (n = 100 + "%") : Math.round(n * 100) / 100 + "%");
```

Comment:
Good use of a ternary expression and rounding! Why assign `100 + "%"` to `n`?

Reply:
this is overkill